plugins/splitbutton/plugin.js

```diff
@@ -36,6 +44,19 @@
 		for ( i in allItems ) {
 			item = allItems[ i ];
 
+			// Split Button can be defined with simpler definition via strings, needed by (#2091).
+			if ( typeof item === 'string' ) {
+				item = CKEDITOR.tools.clone( editor.ui.items[ item ] );
+
+				if ( !item.icon ) {
+					item.icon = item.name || item.command;
```

Comment:
Similar case as before.

Reply:
That is how button plugin works. 
https://github.com/ckeditor/ckeditor-dev/blob/major/plugins/button/plugin.js#L262-L263

```
var name = this.name || this.command,
	iconPath = null,
	overridePath = this.icon;
iconName = name;
```

If there is no `buttonDefinition.icon` property, then icon will fallback to name or command. But `menu plugins` doesn't do that, so in order for menu item to have an icon we need to find that icon same way as `button plugin` does.

Example opening `~/workspace/ckeditor/ckeditor-dev/samples/index.html` and executing following in console:
```
CKEDITOR.instances.editor.ui.get( 'Bold' );
```
Returned object will not have any icon property. Actually when writing this I'm thinking if maybe we should change how button works, so it always have `icon` property when there is icon on button. WDYT @mlewand?

Reply:
As for assigning the `icon` property during button creation - this sounds reasonable if only we started with such approach. Now there might be 3rd party integrations that rely on this property, and rely on it being a "falsy" value. So such change could break some solutions, we don't want to break things.

This rationale sounds reasonable. In this case it make sense to revert this fallback, thanks for clarifying.

Please just drop a short note, or a link to your comment here.

Also make sure to convert the code link above to a perma link, so something like that: https://github.com/ckeditor/ckeditor-dev/blob/6c0469b530cbc767946379e00b5aedcb01e2ba5a/plugins/button/plugin.js#L262-L263